### DISCUSSION TITLE
How to config storybook v6 or v7 in react - 17 project

### DISCUSSION THREAD

**Discussion by timeisnotgood** [storybookjs/storybook#28714] [Help]
### Summary

help me to configure storybook v6 or v7 in react 17 project

### Additional information

i am having a react 17 project and iam trying to configure storybook v6 and v7 in the same , but iam getting lot of diff error like, peer deps err , migrationerror, and storybook not reg as internal or external error

**Comment by AS-EBaiden** [comment]
Following because I'm struggling with something similar. I'm trying to bite the bullet and upgrade from v6 to v8. I'm using this https://storybook.js.org/docs/migration-guide/from-older-version?ref=storybookblog.ghost.io as a point of reference
